core/src/main/java/com/github/weisj/darklaf/ui/rootpane/DarkRootPaneUI.java

```diff
@@ -201,37 +197,30 @@ public void hierarchyChanged(final HierarchyEvent e) {
             return;
         }
         if (parent.getClass().getName().startsWith("org.jdesktop.jdic.tray")
-                || (parent.getClass().getName().compareTo("javax.swing.Popup$HeavyWeightWindow") == 0)) {
+            || (parent.getClass().getName().compareTo("javax.swing.Popup$HeavyWeightWindow") == 0)) {
```

Comment:
Did you mean `.equals("javax.swing.Popup$HeavyWeightWindow")` ?

Reply:
Yes, this has been this way for too long :D